integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -391,36 +396,24 @@ The following is the least-privilege IAM permissions for deploying the connector
 
 ### IAM Execution Permissions
 
-The following is the least privileged IAM permissions for executing the connector.
+The following is the least privileged IAM permissions required for invoking the deployed InfluxDB Timestream connector REST API Gateway.
+
+Replace all items listed below in the IAM policy with values from your AWS account:
+
+- *{region}* &mdash; The AWS region where the InfluxDB Timestream Connector is deployed.
+- *{account-id}* &mdash; The AWS account ID used to deploy the connector.
+- *{api-id}* &mdash; The API ID for the deployed REST API Gateway.
+- *{api-stage-name}* &mdash; The stage name for the deployed REST API Gateway.
+
 
 ```json
 {
     "Version": "2012-10-17",
     "Statement": [
         {
             "Effect": "Allow",
-            "Action": [
```

Comment:
Should we have a section outlining the permissions that the connector will be deployed with? i.e., the permissions deleted here. Or, should users simply review the template?

Reply:
We could add a note stating what permissions are required for ingestion. There are also other policies attached to the Lambda function, for example the `LambdaLogPolicy` and `LambdaSQSDlqPolicy` which are not listed here. If you think it would be useful to add the note about required Timestream permissions here I can, I don't mind either way.

Reply:
Yes, let's do it for better visibility as discussed.